gnome/gstreamer1-gst-plugins-bad/Portfile

```diff
@@ -86,22 +69,7 @@ configure.python            ${prefix}/bin/python${python_branch}
 # requires support for C11 (redefinition of typedef ‘GstTestHTTPSrc’ at elements/test_http_src.c:101)
 compiler.c_standard 2011
 
-configure.args-append       -Ddc1394=disabled \
-                            -Dexamples=disabled \
-                            -Dfaac=disabled \
-                            -Dfdk_aac=disabled \
-                            -Dfrei0r=disabled \
-                            -Dintrospection=enabled \
-                            -Dipcpipeline=disabled \
-                            -Dlv2=disabled \
-                            -Dmpeg2enc=disabled \
-                            -Dmplex=disabled \
-                            -Dopencv=disabled \
-                            -Dopus=disabled \
-                            -Dsrtp=disabled \
-                            -Dstatic=disabled \
-                            -Dhls-crypto=nettle \
-                            -Dapplemedia=disabled
+#configure.args-append       -Dapplemedia=disabled
```

Comment:
Disabled this for the moment as I’ve added something that might work nicer for this, possibly letting us use applemedia for x86_64 while still being disabled for i386 slice.

Reply:
Disabled again as per our conversation.